tests/common/libtest.sh

```diff
@@ -68,6 +68,17 @@ export G_DEBUG=fatal-warnings
 # easily clean up.
 export OSTREE_SYSROOT_DEBUG=mutable-deployments
 
+# See comment in ot-builtin-commit.c and https://github.com/ostreedev/ostree/issues/758
+# Also keep this in sync with the bits in libostreetest.c
+case $(stat -f --printf '%T' /) in
+    overlayfs)
+        echo "overlayfs found; enabling OSTREE_NO_XATTRS"
+        ostree --version
+        export OSTREE_SYSROOT_DEBUG="${OSTREE_SYSROOT_DEBUG},no-xattrs"
```

Comment:
Ahh, I think this is getting overwritten in `setup-session.sh` so isn't taking effect in the daemon.